cypress/e2e/circulation-log/same-date-and-time-format-as-fee-fine-details-page.cy.js

```diff
@@ -0,0 +1,105 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import { TestTypes, DevTeams, Permissions } from '../../support/dictionary';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import TransferAccounts from '../../support/fragments/settings/users/transferAccounts';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import TopMenu from '../../support/fragments/topMenu';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import Users from '../../support/fragments/users/users';
+import UserEdit from '../../support/fragments/users/userEdit';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import SearchResults from '../../support/fragments/circulation-log/searchResults';
+import FeeFineDetails from '../../support/fragments/users/feeFineDetails';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+
+describe('circulation-log', () => {
+  const userData = {};
+  const ownerData = {};
+  const feeFineType = {};
+  const transferAccount = TransferAccounts.getDefaultNewTransferAccount(uuid());
+  let feeFineAccount;
+  let servicePointId;
+
+  before('Create test data', () => {
+    cy.getAdminToken();
+    ServicePoints.getViaApi({ limit: 1 }).then((servicePoint) => {
+      servicePointId = servicePoint[0].id;
+    });
+
+    UsersOwners.createViaApi(UsersOwners.getDefaultNewOwner())
+      .then(({ id, owner }) => {
+        ownerData.name = owner;
+        ownerData.id = id;
+      })
+      .then(() => {
+        TransferAccounts.createViaApi({ ...transferAccount, ownerId: ownerData.id });
+        ManualCharges.createViaApi({
+          ...ManualCharges.defaultFeeFineType,
+          ownerId: ownerData.id,
+        }).then((manualCharge) => {
+          feeFineType.id = manualCharge.id;
+          feeFineType.name = manualCharge.feeFineType;
+          feeFineType.amount = manualCharge.amount;
+        });
+        PaymentMethods.createViaApi(ownerData.id).then(({ name, id }) => {
+          PaymentMethods.name = name;
+          PaymentMethods.id = id;
+        });
+
+        cy.createTempUser([
+          Permissions.circulationLogAll.gui,
+          Permissions.uiUsersfeefinesView.gui,
+          Permissions.uiUsersfeefinesCRUD.gui,
```

Comment:
do you need crud permissions?

Reply:
Apparently, Select 'Fee/fine Details' option from Actions dropdown is visible only when I use crud permission
Without CRUD: 
![image](https://github.com/folio-org/stripes-testing/assets/91249189/42e40e32-4a1d-46f8-b146-563b8f596cf6)
With CRUD: 
![image](https://github.com/folio-org/stripes-testing/assets/91249189/85726fc9-e77f-41cb-ae52-371e45a2d8a0)